support/list_revisions.sh

```diff
@@ -15,4 +15,20 @@ TAGGED_REVISIONS=$(echo "${TAGS}" | grep -E '^v[0-9]+\.[0-9]+\.[0-9]+$')
 # sort tags (which should be in semver-ish format)
 TAGGED_REVISIONS=$(echo "${TAGGED_REVISIONS}" | sort -t "." -k1,1n -k2,2n -k3,3n)
 
-echo "${TAGGED_REVISIONS}"
+# retrieve prefix for branches of prototypes to be listed and published and if
+# it is empty, it can be ignored.
+PROTOTYPE_BRANCHES_REMOTE=${PROTOTYPE_BRANCHES_REMOTE:-""}
+PROTOTYPE_BRANCHES_PREFIX=${PROTOTYPE_BRANCHES_PREFIX:-""}
+
+if [[ -z "${PROTOTYPE_BRANCHES_PREFIX}" || -z "${PROTOTYPE_BRANCHES_REMOTE}" ]]; then
+    # the default values are empty, do nothing
+    echo "${TAGGED_REVISIONS}"
+else
+    # the output of ls-remote is the best for this but doesn't have ready-made
+    # format or pretty-print options. We need to use sed to filter the output
+    # output is the following format:
+    # sha1-hash        refs/heads/branch-name
+    # 012345abcd012345abcd012345abcd012345abcd        refs/heads/branch-name
+    PROTOTYPE_BRANCHES=$(cd "${OSCAL_DIR}"; git ls-remote "${PROTOTYPE_BRANCHES_REMOTE}" "${PROTOTYPE_BRANCHES_PREFIX}*" | sed -n -e "s|^.*\(refs/heads/\)\(${PROTOTYPE_BRANCHES_PREFIX}\)|\2|p")
+    echo "${TAGGED_REVISIONS}" "${PROTOTYPE_BRANCHES}"
+fi
```

Comment:
B: Scripts such as the release asset generator and modeldoc generator rely on the latest release being last (so that it can be extracted with a simple `./list_revisions.sh | tail -n 1`). The script in its current state will break that behavior.

I would request that for consistency we put the prototype revision extraction in a separate script (or directly in the Makefile). This is why for example the Makefile appends `develop`, as the `list_revisions.sh` script's only concern is extracting tagged versions.

Reply:
Maybe part of this PR could be communicating that more clearly, renaming the script to `list_tagged_releases.sh` or something of that sort.

Reply:
So that field appends and not prepends `develop`? OK, let's talk on Gitter about how we order the script invocations. I was not aware of the order requirement, I'm sorry. 🤦

Reply:
My mistake, the Makefile prepends develop to the `REVISIONS` variable, but both of the generation scripts rely on `list_revisions.sh` to get taggged releases. The order of the `REVISIONS` variable in the Makefile does not matter, but the order of `list_revisions.sh`'s output does.

Reply:
https://github.com/usnistgov/OSCAL-Reference/pull/23/commits/2a7ced9018dd1a6bc0a40415aa4792ae55a2a682